shell/platform/embedder/embedder_safe_access.h

```diff
@@ -7,21 +7,25 @@
 
 #include <type_traits>
```

Comment:
This file should probably be called `embedder_macros.h` or something now. It has more than just the `SAFE_ACCESS` macro. We can do this in a separate patch.

Reply:
Easily done in this patch.